src/core_ocean/analysis_members/Registry_high_frequency_output.xml

```diff
@@ -216,92 +216,95 @@
-						<var name="temperatureAvg0700to2000" array_group="activeTracersAvg0700to2000" units="^\circ C" name_in_code="temperatureAvg0700to2000"
-								description="ocean temperature averaged between surface and 100 m"
-						/>
-						<var name="salinityAvg0700to2000" array_group="activeTracersAvg0700to2000" units="PSU" name_in_code="salinityAvg0700to2000"
-								description="salinity averaged between surface and 100 m"
-						/>
-				</var_array>
-				<var_array name="activeTracersAvg2000toBottom" type="real" dimensions="nCells Time">
-						<var name="temperatureAvg2000toBottom" array_group="activeTracersAvg2000toBottom" units="^\circ C" name_in_code="temperatureAvg2000toBottom"
-								description="ocean temperature averaged between surface and 100 m"
-						/>
-						<var name="salinityAvg2000toBottom" array_group="activeTracersAvg2000toBottom" units="PSU" name_in_code="salinityAvg2000toBottom"
-								description="salinity averaged between surface and 100 m"
-						/>
-				</var_array>
+		</var_array>
+		<var_array name="activeTracersAvg0250to0700" type="real" dimensions="nCells Time">
+				<var name="temperatureAvg0250to0700" array_group="activeTracersAvg0250to0700" units="^\circ C" name_in_code="temperatureAvg0250to0700"
+						description="ocean temperature averaged between surface and 100 m"
+				/>
+				<var name="salinityAvg0250to0700" array_group="activeTracersAvg0250to0700" units="PSU" name_in_code="salinityAvg0250to0700"
+						description="salinity averaged between surface and 100 m"
+				/>
+		</var_array>
+		<var_array name="activeTracersAvg0700to2000" type="real" dimensions="nCells Time">
+				<var name="temperatureAvg0700to2000" array_group="activeTracersAvg0700to2000" units="^\circ C" name_in_code="temperatureAvg0700to2000"
+						description="ocean temperature averaged between surface and 100 m"
+				/>
+				<var name="salinityAvg0700to2000" array_group="activeTracersAvg0700to2000" units="PSU" name_in_code="salinityAvg0700to2000"
+						description="salinity averaged between surface and 100 m"
+				/>
+		</var_array>
+		<var_array name="activeTracersAvg2000toBottom" type="real" dimensions="nCells Time">
+				<var name="temperatureAvg2000toBottom" array_group="activeTracersAvg2000toBottom" units="^\circ C" name_in_code="temperatureAvg2000toBottom"
+						description="ocean temperature averaged between surface and 100 m"
+				/>
+				<var name="salinityAvg2000toBottom" array_group="activeTracersAvg2000toBottom" units="PSU" name_in_code="salinityAvg2000toBottom"
+						description="salinity averaged between surface and 100 m"
+				/>
+		</var_array>
 	</var_struct>
 	<streams>
 		<stream name="highFrequencyOutput" type="output"
 				mode="forward;analysis"
 				filename_template="analysis_members/highFrequencyOutput.$Y-$M-$D.nc"
 				filename_interval="01-00-00_00:00:00"
-				output_interval="00-00-01_00:00:00"
+				output_interval="00-00-05_00:00:00"
```

Comment:
@mark-petersen, is this change intentional?  This isn't a whitespace chagne.

Reply:
Right. Updated PR name and description.